var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -782,6 +787,8 @@ def url_for_version(self, version):
         msg="ROCm builds of the GPU package not maintained prior to version 20220623",
     )
     conflicts("+intel", when="%aocc@:3.2.9999", msg="+intel with AOCC requires version 4 or newer")
+    conflicts("+heffte", when="@:20240207", msg="heffte is only supported in 20240207 or newer")
```

Comment:
That version range is inclusive. So you would also be preventing `20240207`.

Reply:
Good catch, fixed!